miso-web/src/main/webapp/scripts/list_transferitem.js

```diff
@@ -6,6 +6,10 @@ ListTarget.transferitem = (function () {
     },
     createBulkActions: function (config, projectId) {
       var actions = [];
+      // pushing a null into actions as people without admin access would otherside have no bulk actions to
+      // interact with table data. Other solutions require refactoring the way bulk actions are given for
+      // the following: samples, libraries, library aliquots and pools.
+      actions.push(null);
```

Comment:
What is the mechanism by which this fixes the issue?

Reply:
It makes the `actions` variable non-empty for all users, which then renders in the regular bulk actions. When `actions` was empty for users not part of a special group, the table would not load regular bulk actions that are needed to interact with table data.

Reply:
Please clarify the comment - sample/library/aliquot/pool actions are added later, after the table's main actions have already been processed. If the main bulk actions are empty, bulk controls (checkboxes, select all/none) are not added. This null causes the bulk actions to be non-empty and is rendered as a separator

The null should only be added if there are no other actions